### PR TITLE
[WIP] Add debug view

### PR DESCRIPTION
**Pull request by camden**
Change notes:
- Removed default Vue components/views
- Add types: MapKey, System, Systems, SaveFile
- Rename `Systems` property of Board to `systems`
- Add example save file
- Add placeholder view and component for the maps view that ingests the example save file

Image of demo: CleanShot 2024-07-28 at 16 16 58@2x

### CODE REVIEW

File: src/Archive.ts [hunk 1]
```diff
@@ -156,10 +161,29 @@ class Board {
...
+  board: {
+    systems: Systems
+  }
+  /**
+   * @param scrap - Array of card objects scrapped from the game (implied cards from fate history are excluded)
+   */
+  scrap: Card[]
+  addedFaithfulActionCards?: Card[]
+  idealMarkers?: string[]
+  act: 1 | 2 | 3
```
Comment (harsch1) [inline src/Archive.ts]: No Act 1 save file since this only supports intermissions

File: .gitignore [hunk 2]
```diff
@@ -27,3 +27,5 @@ coverage
 *.sw?
 
 *.tsbuildinfo
+
+dist/
```
Comment (harsch1) [inline .gitignore]: Why ignore the dist? I wonder if it might break the pages build

**Review by harsch1 — COMMENTED** [review]
Some minor comments but otherwise looks good

File: .gitignore (hunk 2 above)
Comment (camden) [reply .gitignore]: Oops, I didn't realize this was for Pages. Re-added.

File: src/Archive.ts (hunk 1 above)
Comment (camden) [reply src/Archive.ts]: Done.

**Pull request merged**